### DISCUSSION TITLE
App MyVaillant fail

### DISCUSSION THREAD

**Discussion by Enducross** [signalkraft/mypyllant-component#34] [General]
Hi @signalkraft , I have the same installation that has been tested with your integration (Vaillant aroTHERM plus heatpump + sensoCOMFORT VRC 720 + sensoNET VR 921), but when I try to connect the MyVaillant app with my VR921 it tells me that my model is not compatible with the app. I´ve been looking for information about this VR921 and I´ve seen there are five diferent model of VR921
Can you tell me the reference of your VR921?
My VR921 model is: 0020260964

Thankyou.

**Comment by signalkraft** [comment]
Mine is article number 0020260962.

**Reply by Enducross** [reply]
Ok thankyou, I hope vaillant will support my VR921 model on upcoming releases.

**Comment by habeas2023** [comment]
Hi @signalkraft , I use a "ecoTEC plus", sensoHOME VRT 380/2 and sensoNET 921 (Typ 0020260965).
I ordered the sensoNET ...965, because I found the information, the ...962 is for use with aroTHERM and the ...965 for boilers.
Do you have the same informations about the different numbers?
My problem is, I cannot connect the sensoNET 921 Typ ..965 and myVAILLANT-App.
It says "Gateway wird nicht unterstützt. Ihre Anlage ist noch nicht bereit, mit dieser App gesteuert zu werden. Bitte kehren Sie zu Ihrer bisherigen App (sensoAPP oder multiMATIC App) zurück."
My questions are: 
Do I use the right type of sensoNET with my boiler and I still have to wait, until VAILLANT comes with a working app?
Or does other users have the same VAILLANT-devices and everything works?
Thanks for helpful answers.

Solution from VAILLANT arrived today.
VAILLANT writes that first a migration is required, which is apparently carried out by VAILLANT in the background at some point. VAILLANT then sends me an email with a code and only then can I successfully switch from the sensoAPP or multiMATIC app to the myVAILLANT app.
Thanks for listening :-)

13.12.2023
Today, as announced by Vaillant, I received an email with the update path to the myVaillant app. After the firmware update of the sensoNET device started via the app, it now also works with signalkraft's mypyllant component. Currently, 21 entities are now also displayed in my home assistant.
